### PR TITLE
Default api v3

### PR DESCRIPTION
**Pull request by spk**
This use api v3 per default (client will not support v1 with this changes)
v3 support was partial and not released 

http://api.searchmetrics.com/v3/documentation

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-inherit_gem:
```
Comment (maciejmajewski) [inline .rubocop.yml:1]: Warum?

Comment (maciejmajewski) [inline .rubocop.yml:1]: Because of https://github.com/td-berlin/td_critic/pull/14 ?

Comment (spk) [inline .rubocop.yml:1]: Yes and no new release since https://github.com/nevir/rubocop-rspec/releases

Comment (maciejmajewski) [inline .rubocop.yml]: Instead of dropping `td_critic` maybe you can disable only rspec cops?

File: spec/searchmetrics_client/response_spec.rb [hunk 2]
```diff
@@ -12,8 +12,9 @@
     include_context 'with api credentials'
 
     describe '#value' do
-      it 'return array data' do
-        expect(response.value).to be_instance_of Array
+      it 'return hash data' do
+        expect(response.value).to be_instance_of Hash
+        expect(response.value).to eq(balance: 59_981)
```
Comment (JochenFromm) [inline spec/searchmetrics_client/response_spec.rb]: This data ( `balance: 59_981`) does not look like a valid response to me. It is indeed the value the API returns now after this PR, but a valid response for the method `ResearchKeywordsGetListKeywordinfo` if we do a request like 

```
response = SearchmetricsClient::Request.send_request_from_hash(
             endpoint: 'ResearchKeywordsGetListKeywordinfo',
             params: { countrycode: 'PL', keyword: 'ruby' })
response.value
```

should look like

```
[{:cpc=>"2.37", :competition=>"3", :search_volume=>"12366", :trend=>"81:100:100:81:81:81:81:66:81:81:81:81", :ad_budget=>"29307.419999999998", :integration=>"images", :advertiser_count=>"0", :keyword=>"ruby", :year_month=>"201601"}]
```

This is what the old version in the master branch returns and what the API documentation says
http://api.searchmetrics.com/v3/documentation/api-calls/service/ResearchKeywordsGetListKeywordinfo

But now (if we use this branch) for API V3 it returns only a hash with a single value for `balance` like

```
{:balance=>52489}
```

This looks more like a response that our account has exceeded some kind of limit or "balance".

Comment (JochenFromm) [inline spec/searchmetrics_client/response_spec.rb]: It seems to be identical to the value of the available credits

```
response = SearchmetricsClient::Request.send_request_from_hash(
             endpoint: 'AdminStatusGetValueAvailableCredits', 
             params: {})
response.value
```

Comment (spk) [inline spec/searchmetrics_client/response_spec.rb]: Yes you are right this not correct response recorded on the `VCR` cassette, I will look it up.

**Comment by JochenFromm** [comment]
The response from the Searchmetrics API looks much better now! :clap:

**Pull request merged**